modules/topic/index.js

```diff
@@ -0,0 +1,79 @@
+let bot;
+let chunks = null;
+let numChunksAllowed = 16;
+const chunkFileName = 'chunks.json';
+
+module.exports.init = function (b) {
+  bot = b;
+  bot.readDataFile(chunkFileName, (err, jsonData) => {
+    if (err) {
+      bot.log.info('initializing chunks');
+      chunks = {};
+    } else {
+      try {
+        chunks = JSON.parse(jsonData);
+      } catch (ex) {
+        bot.log.warn('Corrupted chunks file! Resetting dict...');
+        chunks = {};
+      }
+    }
+  });
+}
+
+  let store = (chunk, user) => {
+    chunks.chunk = user;
+    bot.writeDataFile(chunkFileName, JSON.stringify(chunks), (err) => {
+      if (err) bot.log.error(`Error writing chunks file: ${err}`);
+    });
+  };
+
+module.exports.commands = {
+  topic(a, parts, b, c, user, channel) {
```

Comment:
It's possible this could be called from privmsg session too (e.g. `/msg bot !topic`).

One solution could be to use the `bot.isChannel(channel)` check and handle it appropriately.